how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-schedule-with-pipeline-parameter-expressions.ipynb

```diff
@@ -30,6 +30,8 @@
    "metadata": {},
    "source": [
     "See below table for supported expressions in schedule. The functions are case sensitive. Suffix `?` means optional parameter, otherwise required.\n",
+    "> Note: Functions startswith `trigger()` are only specific to schedule scenario, because they are about schedule trigger time, and will be treated as raw string in normal pipeline. For rest functions, their behaviors are the same.\n",
```

Comment:
add `or will be treated as raw string in normal pipeline runs` to the end of the description of the two functions.
![image](https://user-images.githubusercontent.com/49483542/157185806-5175e7d2-7879-48fd-9aee-44a70b860f75.png)